pyanaconda/modules/storage/dasd/discover.py

```diff
@@ -58,6 +59,12 @@ def _discover_device(self):
         """Discover the device."""
         # pylint: disable=try-except-raise
         try:
-            blockdev.s390.dasd_online(self._device_number)
-        except blockdev.S390Error as e:
+            rc = execWithRedirect("chzdev",
+                                  ["--enable", "dasd", self._device_number,
+                                   "--active", "--persistent",
+                                   "--yes", "--no-root-update", "--force"])
```

Comment:
I think this is the bit that breaks `DASDTasksTestCase.test_discovery`, but check yourself.

Reply:
Hopefully this fixup of the unit test works: https://github.com/rhinstaller/anaconda/pull/5250/commits/3168bff6c90c4dcb6072d694659a2964fdc9491d